engine/class_modules/priest/sc_priest_discipline.cpp

```diff
@@ -151,25 +151,39 @@ struct power_word_solace_t final : public priest_spell_t
 // Purge the wicked
 struct purge_the_wicked_t final : public priest_spell_t
 {
-  purge_the_wicked_t( priest_t& p, util::string_view options_str )
-    : priest_spell_t( "purge_the_wicked", p, p.talents.discipline.purge_the_wicked )
+  struct purge_the_wicked_dot_t final : public priest_spell_t
   {
-    parse_options( options_str );
-    may_crit  = true;
-    tick_zero = false;
-
-    // Throes of Pain: Spell Direct and Periodic 3%/5% gain
-    apply_affecting_aura( priest().talents.throes_of_pain );
-  }
+    // Manually create the dot effect because "ticking" is not present on
+    // primary spell
+    purge_the_wicked_dot_t( priest_t& p )
+      : priest_spell_t( "purge_the_wicked", p, p.talents.discipline.purge_the_wicked->effectN( 2 ).trigger() )
+    {
+      may_crit = true;
```

Comment:
the spell data has `Periodic Can Crit` so we shouldnt need to set this manually. Would suggest removing and seeing if this is added automatically

Reply:
Removed